### PR TITLE
Add roles for deploying DBT-2 workload

### PR DESCRIPTION
**Pull request by mw2q**

### CODE REVIEW

File: roles/setup_dbt2/tasks/install_packages.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('RedHat', 'RHEL') }}{{ ansible_distribution_major_version }}"
```
Comment (jt-edb) [inline roles/setup_dbt2/tasks/install_packages.yml:5]: I'd suggestion to not change the distribution name, please work with `RedHat`. I know that there are still some areas where we work with `RHEL`, we need to fix that in the future.

File: roles/setup_dbt2/tasks/install_packages.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  yum:
+    name: "{{ dbt2_packages }}"
+  when: os in [ 'CentOS7', 'RHEL7']
+  become: yes
+
+- name: Install packages for DBT-2 (dnf)
+  dnf:
+    name: "{{ dbt2_packages }}"
+  when: os in [ 'CentOS8', 'RHEL8']
+  become: yes
```
Comment (jt-edb) [inline roles/setup_dbt2/tasks/install_packages.yml:17]: Perhaps you could use the `ansible.builtin.package` module for merging both cases into only one task

File: roles/setup_dbt2/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  yum:
+    name: "{{ dbt2_packages_pg }}"
+  when: os in [ 'CentOS7', 'RHEL7']
+  become: yes
+
+- name: Install packages for DBT-2 (server, dnf)
+  dnf:
+    name: "{{ dbt2_packages_pg }}"
+  when: os in [ 'CentOS8', 'RHEL8']
+  become: yes
```
Comment (jt-edb) [inline roles/setup_dbt2/tasks/main.yml:30]: Same comment here around `package` module usage.

File: roles/setup_dbt2/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+- name: Include DBT-2-installation
+  include_tasks: install_dbt2.yml
+
+- name: Install DBT-2 UDF
+  shell:
+    cmd: make USE_PGXS=1 install
+    chdir: /usr/local/src/dbt2/storedproc/pgsql/c
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/usr/pgsql-{{ pg_version }}/bin"
```
Comment (jt-edb) [inline roles/setup_dbt2/tasks/main.yml:40]: `USE_PGXS` could be passed as an environment variable, isn't it?

File: roles/setup_dbt2_client/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# setup_dbt2_client
+
+This role is for setting up a client server for
```
Comment (jt-edb) [inline roles/setup_dbt2_client/README.md:3]: client or server? :)

File: roles/setup_dbt2_client/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+- name: Include DBT-2 kit installation
+  include_role:
+    name: setup_dbt2
+    tasks_from: install_dbt2
+
+- name: Reset the variables based on the user input
+  set_fact:
+      primary_private_ip: ""
+  register: output
```
Comment (jt-edb) [inline roles/setup_dbt2_client/tasks/main.yml:30]: You don't need to register anything at this point.

File: roles/setup_dbt2_driver/tasks/main.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+- name: Include DBT-2 kit installation
+  include_role:
+    name: setup_dbt2
+    tasks_from: install_dbt2
+
+- name: Reset the variables based on the user input
+  set_fact:
+      primary_private_ip: ""
+  register: output
```
Comment (jt-edb) [inline roles/setup_dbt2_driver/tasks/main.yml:30]: Same comment about registering

**Review by jt-edb — COMMENTED** [review]
LGTM. Please see the few minor comments I made.

File: roles/setup_dbt2/tasks/install_packages.yml (hunk 1 above)
Comment (mw2q) [reply roles/setup_dbt2/tasks/install_packages.yml:5]: Ok,fixed.

File: roles/setup_dbt2/tasks/install_packages.yml (hunk 2 above)
Comment (mw2q) [reply roles/setup_dbt2/tasks/install_packages.yml:17]: Ok, consolidated.

File: roles/setup_dbt2/tasks/main.yml (hunk 3 above)
Comment (mw2q) [reply roles/setup_dbt2/tasks/main.yml:30]: Ok, consolidated.

File: roles/setup_dbt2/tasks/main.yml (hunk 4 above)
Comment (mw2q) [reply roles/setup_dbt2/tasks/main.yml:40]: Yes, updated.

File: roles/setup_dbt2_client/README.md (hunk 5 above)
Comment (mw2q) [reply roles/setup_dbt2_client/README.md:3]: Ah, confusing TPC-C terminology.  I'll rephrase.

File: roles/setup_dbt2_client/tasks/main.yml (hunk 6 above)
Comment (mw2q) [reply roles/setup_dbt2_client/tasks/main.yml:30]: Ok, removed.

File: roles/setup_dbt2_driver/tasks/main.yml (hunk 7 above)
Comment (mw2q) [reply roles/setup_dbt2_driver/tasks/main.yml:30]: Ok removed.

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**